share/revng/test/tests/analysis/DetectABI/x86_64/propagate.S.model.yml

```diff
@@ -0,0 +1,24 @@
+#
+# This file is distributed under the MIT LIcense. See LICENSE.md for details.
+#
+
+---
+Functions:
+  - Entry: "0x400660:Code_x86_64"
+    Prototype: "/Types/CABIFunctionType-1"
+Types:
+  - Kind: CABIFunctionType
+    ID: 1
```

Comment:
While any ID works here, for now, ID below a certain number are reserved for `PrimitiveType`s.
This will change, but for now, it's better to here a random `uint64_t`.